src/main/resources/application.conf

```diff
@@ -165,6 +165,9 @@ ergo {
     # Base16 representation of genesis state roothash
     genesisStateDigestHex = "cb63aa99a3060f341781d8662b58bf18b9ad258db4fe88d09f8f71cb668cad4502"
 
+    # Difference between NOW and timestamp of the latest chain update. This helps to discover syncing issues.
+    acceptableChainUpdateDelay = 5m
```

Comment:
Should this be part of chain settings or node settings?

Reply:
Node settings, as this setting is node-specific (can be different on different nodes)